www_src/pages/map/map.jsx

```diff
@@ -29,6 +29,13 @@ var Map = React.createClass({
     var height = 380;
     var gutter = 20;
 
+    if (window.Android) {
```

Comment:
I was thinking it's probably worth replacing the binding mixin stuff with this instead, so we get `getRouteParams` as a method in that mixin